stable/nginx-ingress/values.yaml

```diff
@@ -193,6 +193,27 @@ controller:
       servicePort: 18080
       type: ClusterIP
 
+  ## If controller.stats.enabled = true and controller.metrics.enabled = true, Prometheus metrics will be exported
+  ##
+  metrics:
+    enabled: false
+
+    service:
+      annotations: {}
+      # prometheus.io/scrape: true
```

Comment:
These need to be quoted if they are to work if the comment # is removed

Reply:
@lachie Good catch. Thanks.

@chancez Not sure about this. I'd rather not repeat the key.